capellambse/helpers.py

```diff
@@ -558,7 +558,10 @@ def xtype_of(
 
     plugin_name, plugin_version = symbolic_ns[0][0], symbolic_ns[0][1]
     if not capellambse.check_plugin_version(plugin_name, plugin_version):
-        raise ValueError(f"Not handled version {ns!r}")
+        raise capellambse.UnsupportedVersionError(
+            f"Plugin '{plugin_name}' with unsupported version: "
+            f"'{plugin_version.version}'"
+        )
```

Comment:
Since we already have a custom `UnsupportedVersionError` class, we could also make it a little easier to programmatically handle the error. We could raise it e.g. with:

```python
UnsupportedVersionError(plugin_name, actual_version, supported_min_version, supported_max_version)
```

and then override the `__str__` method in the exception class to transform it into a human readable text like above.